.github/workflows/deploy-github-page.yml

```diff
@@ -141,7 +141,7 @@ jobs:
         shell: bash -l {0}
         run: |
           micromamba activate xeus-lite-host
-          python -m pip install jupyterlite-xeus=4.0.0rc0
+          python -m pip install jupyterlite-xeus==4.0.0rc0
```

Comment:
Guessing we can remove this pin in a day or two once jupyterlite-xeus 4 is out ?